Makefile

```diff
@@ -6,12 +6,13 @@ CFLAGS := -O2 -Wall -g
 LDFLAGS := -static -static-libgcc
 prefix := /usr/local
 
-SRCS := debugcc.c msm8936.c qcs404.c sc8280xp.c sdm845.c sm6115.c sm8350.c sm8450.c
+SRCS := debugcc.c msm8936.c msm8994.c qcs404.c sc8280xp.c sdm845.c sm6115.c sm8350.c sm8450.c
```

Comment:
I can not comment on the commit message itself, so I can comment it here. As you have been mentioning platform names for other platforms, this one is plutonium.

Reply:
I did the because the DTs (and clock drivers for that matter) are now named using these codenames on downstream kernels.. I may add plutonium for completeness though

Reply:
@konradybcio yes, I guessed so. This commit just stood out :-D